Remove terraform state management and unique s3 bucket name (#834)
openfun/marsha: docs/env.md

- Required: Yes
- Default: eu-west-1

#### TF_VAR_s3_bucket_unique_suffix

> A unique suffix added to all S3 bucket name. Once choose this suffix must not be changed, it is not allowed to rename a bucket, terraform will delete the current and create a new one, all the data will be lost for ever. This suffix is needed because a bucket name is unique accross all existing AWS accounts : https://docs.aws.amazon.com/AmazonS3/latest/dev/BucketRestrictions.html

Reviewer: ```suggestion
A unique suffix added to all S3 bucket names. You choose this suffix once and can't change it afterwards as it is not allowed to rename a bucket (terraform would delete the current bucket and create a new one. All your data would be lost for ever). This suffix is needed because a bucket name is unique accross all existing AWS accounts : https://docs.aws.amazon.com/AmazonS3/latest/dev/BucketRestrictions.html
```

> A unique suffix added to all S3 bucket names. You choose this suffix once and can't change it afterwards as it is not allowed to rename a bucket (terraform would delete the current bucket and create a new one. All your data would be lost for ever). This suffix is needed because a bucket name is unique accross all existing AWS accounts : https://docs.aws.amazon.com/AmazonS3/latest/dev/BucketRestrictions.html

Reviewer: ```suggestion
A unique suffix added to all S3 bucket names. You choose this suffix once and can't change it afterwards as it is not allowed to rename a bucket (terraform would delete the current bucket and create a new one. All your data would be lost for ever). This suffix is needed because a bucket name is unique accross all existing AWS accounts: https://docs.aws.amazon.com/AmazonS3/latest/dev/BucketRestrictions.html
```
